data/catalogs/test_data_catalog.py

```diff
@@ -68,19 +73,27 @@ def test_data_catalog(args, datacatalog):
     """Tests the paths of the given data catalog."""
     error_count = 0
     logger.info("Checking paths of data catalog sources")
-    for source in datacatalog.get_source_names():
-        try:
-            logger.info(f"Checking paths of {source}")
-            datacatalog.get_source(source)._resolve_paths()
-        except FileNotFoundError as e:
-            logger.error(f"File not found for dataset source {source}: {e}")
-
-            error_count += 1
-        except ValueError as e:
-            logger.error(
-                f"Something went wrong with creating path string for dataset source {source}: {e}"
+    for source_name, source in datacatalog.__iter__():
+        logger.info(f"Checking paths of {source_name}")
+        if isinstance(source.driver.metadata_resolver, RasterTindexResolver):
+            if not exists(source.full_uri):
+                error_count += 1
+                logger.error(
+                    f"File {source.full_uri} not found for dataset source {source_name}"
+                )
+            continue
```

Comment:
Why does `RasterTindexResolver` need it's own case here?

Reply:
because RasterTindexResolver needs a mask to resolve the paths. Therefore I only check the path of the rastertindex file